exporters/Containerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.access.redhat.com/ubi8/python-39
```

Comment:
use fixed TAG?

Reply:
did not like the available options, going to stick with latest
![Screenshot from 2023-05-31 16-58-47](https://github.com/dora-metrics/pelorus/assets/66965232/25685dd3-e234-452f-ba0d-bed7cec11536)